app/schemas/submission.py

```diff
@@ -3,12 +3,16 @@
 from .user import StudentSchema
 from .assignment import AssignmentSchema
 
-class SubmissionSchema(BaseModel):
+class DatabaseSubmissionSchema(BaseModel):
```

Comment:
Why the separation between `DatabaseSubmissionSchema` and `SubmissionSchema` here? Why not just put the `active` flag on the original `SubmissionSchema` object?

Reply:
It's the same concept as StudentAssignmentSchema and InstructorAssignmentSchema. I would eventually like to go through and refactor all the base schemas into `Database{SchemaName}` so that it's more clear.

Basically, active has to be computed using a query, so it has to be added in when creating SubmissionSchema, whereas DatabaseSubmissionSchema can be created directly from a SubmissionModel.